neurokit2/ecg/ecg_phase.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import pandas as pd
+
+from ..signal import signal_phase
+
+
+
+def ecg_phase(ecg_cleaned, rpeaks=None, delineate_info=None, method='dwt',  sampling_rate=None):
+    """Compute cardiac phase (for both atrial and ventricular).
+
+    Finds the cardiac phase, labelled as 1 for systole and 0 for diastole.
```

Comment:
I'd add a bit of info about what's the difference between these two types of phase and also which one is the "major" one (the most studied in regards to cognition for instance)